buildSrc/src/main/kotlin/de/itemis/mps/buildbackends/Versioning.kt

```diff
@@ -0,0 +1,11 @@
+package de.itemis.mps.buildbackends
+
+fun computeVersionSuffix(): String {
+    val buildCounterStr = System.getenv("BUILD_COUNTER")
+    if (buildCounterStr.isNullOrEmpty()) {
+        return "-SNAPSHOT"
+    } else {
+        val gitCommitHash = getCommandOutput("git", "rev-parse", "--short=7", "HEAD")
+        return ".$buildCounterStr.$gitCommitHash"
```

Comment:
Shouldn't the "." be removed when the version is a snapshot version?

Otherwise, for example, the project loader version might look like

`2.0.0.-SNAPSHOT.a29d777` instead of `2.0.0-SNAPSHOT.a29d777`